package.json

```diff
@@ -53,10 +53,10 @@
     ]
   },
   "scripts": {
-    "testbuild": "node bin/generate.js -t -n solo -d ./auro-test",
+    "testbuild": "node bin/generate.js -t -N Han -n Solo -P @mypackage/ -d ./auro-test",
```

Comment:
Was the "S" in solo intended to be capitalized?

Reply:
Yes, there is a function to remove the capitalization.